### ISSUE TITLE
Uncaught Exception: Ran out of stack space trying to match the regular expression.

### ISSUE THREAD

**Issue by bhajneet** [mawww/kakoune#737]
```
uncaught exception (N5boost16exception_detail10clone_implINS0_19error_info_injectorISt13runtime_errorEEEE):
Ran out of stack space trying to match the regular expression.
```

after trying to "sudo kak /etc/fstab"

**Comment by lenormf** [comment]
Fix pending in #735.

**Comment by mawww** [comment]
#735 has been merged.
